src/app/test-results/test-results-list/test-results-list.component.html

```diff
@@ -0,0 +1,50 @@
+<div id="test-results-list">
+  <p-table [value]="summaries">
+    <ng-template pTemplate="header">
+      <tr>
+        <th class="col-id">ID</th>
+        <th class="col-name">FILENAME</th>
+        <th class="col-status">STATUS</th>
+      </tr>
+    </ng-template>
+    <ng-template pTemplate="body" let-summary>
+      <tr role="button" (click)="this.toggleDetails(summary)">
+        <td class="col-id">{{summary.testResultSummaryId}}</td>
+        <td class="col-name">{{summary.fileName}}</td>
+        <td class="col-status">{{summary.failed ? "Failed" : "Passed"}}</td>
```

Comment:
For a better overview, I think listing how many has passed or failed here would be a great addition. Perhaps also using the same icons you used in the build list.

And for better visual indication, you could gray out the failed icon if there are no failing tests, and gray out the passed icon if there are no passing tests for that file, etc.